src/test/resources/robots/conflicting-rules.txt

```diff
@@ -0,0 +1,16 @@
+User-Agent: testy
+Disallow: /foo
+
+User-Agent: curl
+Disallow: /
+
+User-Agent: wget
+Allow: /
+Disallow: /*
```

Comment:
The [Google spec](https://developers.google.com/search/docs/advanced/robots/robots_txt#url-matching-based-on-path-values) lists `/*` as equivalent of `/`. Of two equivalent rules the less restrictive is chosen.

Reply:
Well, they say that, but their code doesn't actually do that at all. I tested this with some examples in the wild, and my own concocted examples. Given / and /*, they'll assuming /* is of higher priority (because it also matches, and pattern length is greater).

Reply:
The RFC doesn't specify that `/*` needs to be handled in a special way, so I think this version is more RFC-compliant (and is more consistent with Google-code, rather than their docs)